apps/src/code-studio/pd/application/principalApproval1920/PrincipalApproval1920Component.jsx

```diff
@@ -12,23 +12,36 @@ import {styles} from '../teacher1920/TeacherApplicationConstants';
 const MANUAL_SCHOOL_FIELDS = ['schoolName', 'schoolAddress', 'schoolCity',
   'schoolState', 'schoolZipCode', 'schoolType'];
 const RACE_LIST = ['white', 'black', 'hispanic', 'asian', 'pacificIslander', 'americanIndian', 'other'];
-const REQUIRED_SCHOOL_INFO_FIELDS = ['school', 'totalStudentEnrollment',
-  'freeLunchPercent', ...RACE_LIST, 'committedToMasterSchedule', 'hoursPerYear',
-  'termsPerYear', 'replaceCourse', 'committedToDiversity', 'understandFee', 'payFee'
+const REQUIRED_SCHOOL_INFO_FIELDS = ['goingToTeach', 'school', 'totalStudentEnrollment',
+  'freeLunchPercent', ...RACE_LIST, 'committedToMasterSchedule', 'replaceCourse', 'committedToDiversity',
+  'understandFee', 'payFee', 'howHeard'
 ];
+// Since the rails model allows empty principal approvals as placeholders, we require these fields here
```

Comment:
Is the only reason these are here instead of required on the model to allow empty placeholder responses? 
And the placeholder exists so we know whether a principal approval email has been sent (so we don't send another one)?

I think we can clean this up. After my pending [email PR](https://github.com/code-dot-org/code-dot-org/pull/24950) I'd prefer leveraging the new `Email` queue for this purpose and simplifying this form logic. Since that's not merged yet we can do it as a followup (no need to hold up this PR now), but I think we can simplify this quite a bit by removing placeholder support. Looking at the email queue would more directly answer the question we want to know anyway, if I'm understanding this correctly.

Thoughts?

Reply:
That's exactly why they are there, and looking at the new email queue definitely makes more sense than the placeholder system. I'll put a card on the backlog to clean this up. thanks!